include/aspect/postprocess/composition_BPL_statistics.h

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
unused files (left from early version) removed. Replaced BPL appeared in all other places